css/style.css

```diff
@@ -750,10 +763,10 @@ textarea {
   top: 100%;
   visibility: hidden;
   opacity: 0;
-  transition: all .3s;
-  box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
-  -webkit-box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
+  transition: all 0.3s;
   -moz-box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
```

Comment:
you must first define" -moz-box-shadow" , "-webkit-box-shadow"  before this "box-shadow" to work properly